tools/mtouch/Assembly.cs

```diff
@@ -283,6 +283,7 @@ public void CreateAOTTask (Abi abi)
 
 			var aotCompiler = Driver.GetAotCompiler (App, Target.Is64Build);
 			var aotArgs = Driver.GetAotArguments (App, assembly_path, abi, build_dir, asm_output ?? other_output, llvm_aot_ofile, data);
+			Driver.Log (1, "AOT compiler call: {0} {1}", aotCompiler, aotArgs);
```

Comment:
Don't we already show this?

From a sample build I just did:

```
...
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/unifiedtestapp.aotdata.arm64,static,asmonly,direct-icalls,full,soft-debug,dwarfdebug,no-direct-calls,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/unifiedtestapp.exe.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/unifiedtestapp.exe"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.Core.aotdata.arm64,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,direct-pinvoke,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.Core.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/System.Core.dll"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/Xamarin.iOS.aotdata.arm64,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/Xamarin.iOS.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/Xamarin.iOS.dll"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.aotdata.arm64,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,direct-pinvoke,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/System.dll"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.Net.Http.aotdata.arm64,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,direct-pinvoke,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/System.Net.Http.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/System.Net.Http.dll"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm64-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=arm64-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/mscorlib.aotdata.arm64,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,direct-pinvoke,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/arm64/mscorlib.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/64/3-Build/mscorlib.dll"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/32/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=armv7-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/armv7/unifiedtestapp.aotdata.armv7,static,asmonly,direct-icalls,full,soft-debug,dwarfdebug,no-direct-calls,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/armv7/unifiedtestapp.exe.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/32/3-Build/unifiedtestapp.exe"
MONO_PATH=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/32/3-Build /work/maccore/issue-5888-older-simulators/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/bin/arm-darwin-mono-sgen --debug -O=gsharedvt -O=-float32  --aot=mtriple=armv7-ios,data-outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/armv7/mscorlib.aotdata.armv7,static,asmonly,direct-icalls,full,dwarfdebug,no-direct-calls,direct-pinvoke,outfile=/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/armv7/mscorlib.dll.s "/Users/rolf/Projects/TestApp/obj/iPhone/Debug/mtouch-cache/32/3-Build/mscorlib.dll"
...
```

Reply:
afaik we have those
it's even repeated (a 2nd time) if the AOT compiler returns an error